src/provider/FDC3.ts

```diff
@@ -360,8 +402,8 @@ export class FDC3 {
         });
     }
 
-    private async sendContext(context: ContextBase): Promise<void> {
-        return fin.InterApplicationBus.publish('context', context);
+    private async sendContext(identity: Identity, context: ContextBase): Promise<void> {
+        return fin.InterApplicationBus.send(identity, 'context', context);
```

Comment:
Are we going to switch this to use an IAB channel? To avoid adding extra work to SERVICE-417?

Reply:
Figured leaving this as is for now was going to conflict less with Theo's work than changing to use channels. It's not a new use of IAB, can changing it here would necessitate changing it at the receiver end (right?), which would also conflict